x/ecocredit/server/core/msg_create_class_test.go

```diff
@@ -212,41 +220,48 @@ func (s *createClassSuite) ExpectTheResponse(a gocuke.DocString) {
 }
 
 func (s *createClassSuite) createClassExpectCalls() {
-	var allowlistEnabled bool
-	var allowedClassCreators []string
-	var creditClassFee sdk.Coins
-
-	s.paramsKeeper.EXPECT().
-		Get(s.sdkCtx, core.KeyAllowlistEnabled, &allowlistEnabled).
-		Do(func(ctx sdk.Context, key []byte, allowlistEnabled *bool) {
-			*allowlistEnabled = s.params.AllowlistEnabled
-		}).
-		AnyTimes() // not expected on failed attempt
+	err := s.stateStore.AllowListEnabledTable().Save(s.ctx, &api.AllowListEnabled{
+		Enabled: s.allowlistEnabled,
+	})
+	require.NoError(s.t, err)
```

Comment:
The idea with `createClassExpectCalls` is that it includes the expected mock calls. We no longer have expected mock calls for the core params now that we are using ORM params for core params. This should be included in the `AllowlistEnabled` step.